tools/crm_diff.c

```diff
@@ -296,6 +296,8 @@ main(int argc, char **argv)
     gchar **processed_args = pcmk__cmdline_preproc(argv, "nopNO");
     GOptionContext *context = build_arg_context(args);
 
+    int rc = 0;
```

Comment:
You probably meant to set it to pcmk_rc_ok here?  Actually, maybe it would be worth adding an additional patch to convert apply_patch and generate_patch to use the new standard return codes.

Reply:
why not